app/views/govuk_component/taxonomy_sidebar.raw.html.erb

```diff
@@ -1,20 +1,36 @@
 <% if local_assigns[:items] && !items.blank? %>
   <aside class='govuk-taxonomy-sidebar' data-module='track-click' role='complementary'>
-    <% items.each_with_index do |item, item_index| %>
+    <% items.sort_by { |item| item[:title] }.each_with_index do |item, item_index| %>
```

Comment:
What do you think of moving this logic to `govuk_navigation_helpers` instead? We already do some sorting in there for breadcrumbs, which means the govuk components become dummy.

We could, for example, move the sorting from this line: https://github.com/alphagov/govuk_navigation_helpers/blob/master/lib/govuk_navigation_helpers/content_item.rb#L23 into this line: https://github.com/alphagov/govuk_navigation_helpers/blob/master/lib/govuk_navigation_helpers/content_item.rb#L34

This way, we wouldn't need to sort data here.